src/resources/resources/assets/waila/lang/ru_ru.json

```diff
@@ -1,115 +1,216 @@
 {
-  "tooltip.waila.crop_growth"                 : "Зрелость",
-  "tooltip.waila.crop_mature"                 : "Созрело",
-  "tooltip.waila.empty"                       : "Пусто",
-  "tooltip.waila.delay"                       : "Задержка",
-  "tooltip.waila.mode"                        : "Режим",
-  "tooltip.waila.mode_comparator"             : "Сравнение",
-  "tooltip.waila.mode_subtractor"             : "Вычитание",
-  "tooltip.waila.state"                       : "Состояние",
-  "tooltip.waila.state_on"                    : "Вкл.",
-  "tooltip.waila.state_off"                   : "Выкл.",
-  "tooltip.waila.power"                       : "Мощность",
-  "tooltip.waila.sneak_for_details"           : "Пригнитесь для подробностей",
-  "tooltip.waila.honey_level"                 : "Уровень мёда",
-
-  "tooltip.waila.instrument.harp"             : "Арфа",
-  "tooltip.waila.instrument.basedrum"         : "Большой барабан",
-  "tooltip.waila.instrument.snare"            : "Малый барабан",
-  "tooltip.waila.instrument.hat"              : "Хай-хэт",
-  "tooltip.waila.instrument.bass"             : "Басс",
-  "tooltip.waila.instrument.flute"            : "Флейта",
-  "tooltip.waila.instrument.bell"             : "Колокольчики",
-  "tooltip.waila.instrument.guitar"           : "Гитара",
-  "tooltip.waila.instrument.chime"            : "Чаймс",
-  "tooltip.waila.instrument.xylophone"        : "Ксилофон",
-  "tooltip.waila.instrument.iron_xylophone"   : "Вибрафон",
-  "tooltip.waila.instrument.cow_bell"         : "Коровий колокольчик",
-  "tooltip.waila.instrument.didgeridoo"       : "Диджериду",
-  "tooltip.waila.instrument.bit"              : "Бит",
-  "tooltip.waila.instrument.banjo"            : "Банджо",
-  "tooltip.waila.instrument.pling"            : "Палочки",
```

Comment:
Missing translations:
```
  "tooltip.waila.instrument.zombie"                             : "Zombie",
  "tooltip.waila.instrument.skeleton"                           : "Skeleton",
  "tooltip.waila.instrument.creeper"                            : "Creeper",
  "tooltip.waila.instrument.dragon"                             : "Ender Dragon",
  "tooltip.waila.instrument.wither_skeleton"                    : "Wither Skeleton",
  "tooltip.waila.instrument.piglin"                             : "Piglin",
  "tooltip.waila.instrument.none"                               : "None",
```

Reply:
Already added and translated these lines.